test/UI/page_object/forms_page.py

```diff
@@ -1,61 +1,135 @@
+        self.send_keys(self.PHONE_FIELD, phone)
+
+    @handle_none_argument
+    def enter_date_of_birth(self, date_of_birth: str) -> None:
+        self.send_keys(self.DATE_OF_BIRTH_FIELD, date_of_birth)
+
+    @handle_none_argument
+    def enter_subjects(self, subjects: str) -> None:
+        self.send_keys(self.SUBJECTS_FIELD, subjects)
+
+    @handle_none_argument
+    def click_hobby(self, hobbies: List[Hobby]) -> None:
+        for hobby in hobbies:
+            self.click(self.replace_placeholders_in_locator(self.HOBBY_CHECKBOX, hobby))
+
+    @handle_none_argument
+    def enter_current_address(self, address: str) -> None:
+        self.send_keys(self.CURRENT_ADDRESS_FIELD, address)
+
+    @handle_none_argument
+    def enter_state(self, state) -> None:
+        pass
+
+    @handle_none_argument
+    def enter_city(self, city) -> None:
+        pass
+
+    def click_submit_button(self) -> None:
         self.driver.execute_script("window.scrollTo(0, document.body.scrollHeight)")
-        self.click(forms_page_locators.SUBMIT_BUTTON)
+        self.click(self.SUBMIT_BUTTON)
 
-    def check_visibility_of_alert_symbol_on_first_name_field(self):
-        value = self.value_of_css_property(forms_page_locators.FIRST_NAME_FIELD, "background-image")
+    def check_visibility_of_alert_symbol_on_first_name_field(self) -> bool:
+        value = self.value_of_css_property(self.FIRST_NAME_FIELD, "background-image")
         if value:
             return True
 
-    def check_visibility_of_alert_symbol_on_last_name_field(self):
-        value = self.value_of_css_property(forms_page_locators.LAST_NAME_FIELD, "background-image")
+    def check_visibility_of_alert_symbol_on_last_name_field(self) -> bool:
+        value = self.value_of_css_property(self.LAST_NAME_FIELD, "background-image")
         if value:
             return True
 
-    def check_visibility_of_alert_symbol_on_mobile_field(self):
-        value = self.value_of_css_property(forms_page_locators.PHONE_FIELD, "background-image")
+    def check_visibility_of_alert_symbol_on_mobile_field(self) -> bool:
+        value = self.value_of_css_property(self.PHONE_FIELD, "background-image")
         if value:
             return True
 
     # submitted data window
-    def wait_until_submitted_data_window_opens(self):
-        WebDriverWait(self.driver, 5).until(
-            EC.presence_of_element_located(forms_page_locators.WINDOW_OF_SUBMITTED_DATA))
-
-    def get_name_on_window_of_submitted_data(self):
-        name = self.get_text(forms_page_locators.NAME_ON_WINDOW_OF_SUBMITTED_DATA)
-        return name
-
-    def get_phone_on_window_of_submitted_data(self):
-        phone = self.get_text(forms_page_locators.PHONE_ON_WINDOW_OF_SUBMITTED_DATA)
-        return phone
-
-    def get_gender_on_window_of_submitted_data(self):
-        gender = self.get_text(forms_page_locators.GENDER_ON_WINDOW_OF_SUBMITTED_DATA)
-        return gender
+    def wait_until_submitted_data_window_opens(self) -> None:
```

Comment:
Can you move this method to `SubmittedDataForm` with name `wait_for_shown`? Bkz it is related to the form, rather to the page. In the test it would be something like:
```
submitted_data_form = forms_page.SubmittedDataForm(driver)
submitted_data_form.wait_for_shown();
```

Reply:
done